src/Middleware/CensorRequestBodyFields.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Facade\FlareClient\Middleware;
+
+use Facade\FlareClient\Report;
+
+class CensorRequestBodyFields
+{
+    protected $fieldNames = [];
+
+    public function __construct(array $fieldNames)
```

Comment:
Should we allow a closure here?

Reply:
Let's start simple with only an array of fields here.

Can you share a scenario where a closure could be handy?

Reply:
I think it could be handy to completely change the request made to flare, remove context items, remove server information, ...

But that's maybe a bit out of scope for this PR